### ISSUE TITLE
Footnotes broken

### ISSUE THREAD

**Issue by 1wheel** [kjhealy/plain-text.co#9]
Clicking on the footnotes only adds `#fn3` to the URL. 

The footnote text in the [rmd](https://github.com/kjhealy/plain-text.co/blob/master/index.Rmd) ("As opposed to multi-tasking in the less-interesting") isn't present in the rendered html.

**Comment by dajare** [comment]
Yeah ... still broken. A wee shame, as it's not a great advert for "[plain text](https://plain-text.co/#fn1)", is it? `:/`

If one view's source, it's clear that the footnote text is simply AWOL. Has something gone wrong in the pandoc generation (presumably) of the HTML?
